src/operation-node/query-node.ts

```diff
@@ -19,6 +22,8 @@ type HasJoins = { joins?: ReadonlyArray<JoinNode> }
 type HasWhere = { where?: WhereNode }
 type HasReturning = { returning?: ReturningNode }
 
+export type HasExplain = { explain?: ExplainNode }
```

Comment:
Why export?

Reply:
Haven't updated after the other types were un-exported.